docs/pages/guides/config-plugins.mdx

```diff
@@ -841,7 +840,7 @@ For managed users, we can expose this functionality (safely!) via an Expo config
 `expo-custom/app.plugin.js`
 
 ```js
-const { AndroidConfig, withStringsXml } = require('@expo/config-plugin');
+const { AndroidConfig, withStringsXml } = require('expo/config-plugin');
```

Comment:
Shouldn't this be `expo/config-plugins` (-s)?

Reply:
yes

Reply:
```suggestion
const { AndroidConfig, withStringsXml } = require('expo/config-plugins');
```

Reply:
(it looks like the typo was already there so it wasn't picked up from find/replace)